client/dashboard/sites/staging-site-delete-modal/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { useMutation } from '@tanstack/react-query';
+import { useRouter } from '@tanstack/react-router';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Button,
+	Modal,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { stagingSiteDeleteMutation } from '../../app/queries/staging-sites';
+import type { Site } from '../../data/types';
+
+export default function StagingSiteDeleteModal( {
+	site,
+	onClose,
+}: {
+	site: Site;
+	onClose: () => void;
+} ) {
+	const router = useRouter();
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	const parentSiteId = site.options?.wpcom_production_blog_id;
+
+	const mutation = useMutation(
+		parentSiteId
+			? stagingSiteDeleteMutation( site.ID, parentSiteId )
+			: {
+					mutationFn: async () => {
+						throw new Error( 'Parent site not found' );
+					},
+			  }
+	);
+
+	const handleDelete = () => {
+		if ( ! parentSiteId ) {
+			createErrorNotice( __( 'Cannot delete staging site: parent site not found' ), {
+				type: 'snackbar',
+			} );
+			onClose();
+			return;
+		}
+
+		mutation.mutate( undefined, {
+			onSuccess: () => {
+				router.navigate( { to: '/sites' } );
+				createSuccessNotice( __( 'Staging site has been deleted.' ), { type: 'snackbar' } );
```

Comment:
We want to make the tone consistent. See ref:

- https://github.com/Automattic/wp-calypso/pull/104252

Could we update it  

```suggestion
				createSuccessNotice( __( 'Staging site deleted.' ), { type: 'snackbar' } );
```

Reply:
Thanks for the pointer, adjusted in https://github.com/Automattic/wp-calypso/pull/104443/commits/6f881f9c36865435f9fc52ba7f1cace4dbe5c432